File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/CustomExpiredJwtException.java
package com.dogGetDrunk.meetjyou.common.exception.business;

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode;

public class CustomExpiredJwtException extends CustomJwtException {

    private String value;

    public CustomExpiredJwtException(String value) {
        super(value, ErrorCode.EXPIRED_TOKEN);
    }
}

File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/jwt/CustomExpiredJwtException.kt
package com.dogGetDrunk.meetjyou.common.exception.business.jwt

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode

class CustomExpiredJwtException(
    value: String?,
    message : String? = null,
) : CustomJwtException(ErrorCode.EXPIRED_TOKEN, value, message)
